pages/country.js

```diff
@@ -43,6 +44,49 @@ const AnimatedFlex = styled(Flex)`
 const AnimatedHeading = styled(Heading)`
   transition: all 0.5s ease;
 `
+const IntlHead = ({
```

Comment:
Let's move this component into its own file under `components/country/`

Reply:
Ok, I'll do that and send another PR.